src/clients/UBAClient/UBAClientAbstract.ts

```diff
@@ -142,23 +142,23 @@ export abstract class BaseUBAClient {
 
   /**
    * Compute the entire system fee for a given amount. The system fee is the sum of the LP fee and the balancing fee.
-   * @param depositChain The chainId of the deposit
-   * @param refundChain The chainId of the refund
+   * @param depositChainId The chainId of the deposit
+   * @param destinationChainId The chainId of the transaction
    * @param spokePoolToken The token to get the system fee for
    * @param amount The amount to get the system fee for
    * @param hubPoolBlockNumber The block number to get the system fee for
    * @returns The system fee for the given token on the given chainId at the given block number
    */
   public async computeSystemFee(
-    depositChain: number,
-    refundChain: number,
+    depositChainId: number,
+    destinationChainId: number,
     spokePoolToken: string,
     amount: BigNumber,
     hubPoolBlockNumber: number
   ): Promise<SystemFeeResult> {
     const [lpFee, { balancingFee: depositBalancingFee }] = await Promise.all([
-      this.computeLpFee(spokePoolToken, depositChain, refundChain, amount),
-      this.computeBalancingFee(spokePoolToken, amount, hubPoolBlockNumber, depositChain, UBAActionType.Deposit),
+      this.computeLpFee(spokePoolToken, depositChainId, destinationChainId, amount),
+      this.computeBalancingFee(spokePoolToken, amount, hubPoolBlockNumber, destinationChainId, UBAActionType.Deposit),
```

Comment:
Why did you change this line? I thought this balancing fee would be based on the deposit chain only?

Reply:
This was a reversion from testing. Resolved here: https://github.com/across-protocol/sdk-v2/pull/248/commits/ee8bab88bf45089a1fbaa2d03fb84c11c59e922a